Mvp dev more

yzhg-ux opened the pull request.

tancolo committed:
第14课，对Rxjava+retroit 使用

tancolo committed:
第15课, 添加CompositeSubscription 管理多个subscription

tancolo committed:
第15课， 对Book模块使用Rxjava

tancolo committed:
第16课，添加导航栏，blog， aboutme Fragment， 内容待实现

tancolo committed:
第17课，添加json数据到github

tancolo committed:
第17课，修改github raw上的 json数据格式

tancolo committed:
第17课，完成Blog内容显示，内容从github raw中获取，并将自定义View移到ui目录

tancolo committed:
第17课，完成Aboutme内容显示

tancolo committed:
第18课，继承ShareSdk. 如何继承参考官网

tancolo committed:
第18课，集成ShareSdk. 如何集成参考官网

tancolo committed:
第18课，按照社交平台要求，需要新的启动icon以及尺寸大小

tancolo committed:
第18课，解决git push冲突

tancolo committed:
第18课，修改包名 idouban -> ishuying

tancolo committed:
第18课，QQ分享完成

tancolo committed:
第18课，QQ登录测试成功

tancolo committed:
第18课，改写ShareSdk登录方式，使用rxjava包装callback

tancolo committed:
第20课， 实现一次登录，各种响应。以及数据的持久化sp

tancolo committed:
第20课，实现退出登录后，清楚本地缓存以及变换UI，通知HomeActivity 登录退出

tancolo committed:
第20课，实现自动登录。 从文件恢复User数据，并用于全局。 同时修改其他文件中的Log级别

tancolo committed:
Bug修复
1. 导航栏图片能正确进入Aboutme Fragment
2. 所有Activity都是竖屏
3. HomeActivity & LoginActivity 其中模式修改

更改:
使用RxBroadcast 替换原生接收Local广播方式

tancolo committed:
[Feature]
使用Behavior方式实现上下滑动显示隐藏FAB

tancolo committed:
[Modify]
MovieDetailActivity 分享操作在FAB， 滑动隐藏显示

tancolo committed:
[Feature]
1. 修改主题色调
2.电影，书籍详情页面 加入全局返回 <-

tancolo committed:
[Release to google play]
1. 删除多余权限
2. 修复release 编译问题

tancolo committed:
update readme

tancolo committed:
[Feature]
修改默认图片

tancolo committed:
[重构] 升级到rxjava2 涉及的修改有 Retrofit2 Adapter， 网络Presenter获取数据Observable等

tancolo committed:
[update] 更新 readme.md

tancolo committed:
[update] 更新gradle插件相关内容，请使用AS3.0+开发环境

tancolo commented:
Thank you for your submitting PR, but this seems to be a lot of changes, github prompts too many conflicts, please submit the PR on the mvp-dev-more branch latest code, and write your description.
Thank you for your cooperation.

Commit:
[update] 更新gradle插件到3.4.2, wrapper gradle到5.1.1-all.zip  解决编译问题
可以不使用RetroLambda, 迁移方式如下
https://developer.android.com/studio/write/java8-support?utm_source=android-studio

Commit:
[Bug fix]
解决数据源API失效问题

tancolo committed:
Update README.md
